Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -989,6 +1125,28 @@ namespace O3DE::ProjectManager
         UpdateIfCurrentScreen();
     }
 
+    void ProjectsScreen::ProjectExportDone(bool success)
+    {
+        ProjectInfo currentExportProject;
+        if (!success)
+        {
+            currentExportProject = m_currentExporter->GetProjectInfo();
+        }
+
+        m_currentExporter.reset();
+
+        if (!m_exportQueue.empty())
+        {
+            while (!StartProjectExport(m_exportQueue.front()) && m_exportQueue.size() > 1)
+            {
+                m_exportQueue.pop_front();
+            }
+            m_exportQueue.pop_front();
+        }
+
+        UpdateIfCurrentScreen();
```

Comment:
Do we also want to prompt the user to open the 'file location' of the exported projects when the project export is successful?

Reply:
I was thinking this too. Do we want to handle this from Project Manager, or from the export scripts?

Reply:
I'm thinking it might be easier from export script, b/c each platform has a different output path it could go to.

Reply:
No, it should be from the project manager. You don't want CLIs to pop up any message box that prompts because its used pretty often from automated scripts, and any UI will block the process.

Reply:
I'm handling it as an export config option, so that each export script can choose the appropriate output folder. new commit will show the changes

Reply:
and no, not in all cases can you easily get the relevant output path. This would be true if it were just PC export, but android and ios have specific output paths that can differ, either by o3de toolchain dictating it, or by user preference. It's much messier trying to coordinate that from the Project Manager, as there would need to be a lot of logic in projectinfo just to check which output path you're referring to.